pixel.js

```diff
@@ -78,7 +78,7 @@ async function processCommand( type, opts ) {
 		// Execute main.js.
 		await batchSpawn.spawn(
 			'docker-compose',
-			[ 'exec', 'mediawiki', '/src/main.js', JSON.stringify( opts ) ]
+			[ 'exec', '-T', 'mediawiki', '/src/main.js', JSON.stringify( opts ) ]
```

Comment:
I think this may be causing the script to ignore presses to `ctrl-c` on one of my laptops, however it seems to respond fine on my other computer so I'm not sure. I think it also disables color output at least for that script. Can you replicate?

Also, what was the original problem? Running the cron job on cloud vps?

Reply:
Try running in a cronjob context locally and you should be able to replicate.